### PR TITLE
added functionality to input img source link

### PR DESCRIPTION
**Pull request by valeriegc**
Added a functionality for user to put in their img source, when adding image from their local computer.

### CODE REVIEW

File: src/routes/Canvas.svelte [hunk 1]
```diff
@@ -76,14 +76,23 @@
 				</div>
 			{/each}
 		</div>
+		<div class="inputWrap">
+			Please input the source for your image here:
+			<input bind:value={$userProvidedLink} />
+		</div>
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: If the input field is placed after the image and a user uploads a large image, isn't there a risk that he will miss it?

File: src/routes/Canvas.svelte [hunk 2]
```diff
@@ -76,14 +76,23 @@
...
 	</div>
 {/if}
 
 <style>
+	.inputWrap {
+		margin-top: 30px;
+		padding-left: 10px;
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: Why did you need to add a padding here?

File: src/routes/Canvas.svelte [hunk 3]
```diff
@@ -76,14 +76,23 @@
 				</div>
 			{/each}
 		</div>
+		<div class="inputWrap">
+			Please input the source for your image here:
+			<input bind:value={$userProvidedLink} />
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: Do you want to add some styles, so its more consistent with the rest of the app?

File: src/routes/Canvas.svelte (hunk 1 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: good point Alex! thank you. I have added it now above.

File: src/routes/Canvas.svelte (hunk 2 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: removed it.

File: src/routes/Canvas.svelte [hunk 4]
```diff
@@ -47,6 +47,13 @@
 
 {#if $files?.[0]}
 	<div class="completeWrap">
+		<div class="inputWrap">
+			Please input the source for your image here:
+			<input
+				bind:value={$userProvidedLink}
+				style="margin-left:10px;border-radius:5px; border-color:darkblue; color:darkblue"
```
Comment (AlexanderKolberg) [inline src/routes/Canvas.svelte]: Screenshot 2023-05-02 at 18 48 26

The input has some interesting styling.  Gets fixed if you use the shorthand border property

**Review by AlexanderKolberg — APPROVED** [review]
LGTM

File: src/routes/Canvas.svelte (hunk 4 above)
Comment (valeriegc) [reply src/routes/Canvas.svelte]: Fixed.
